configuration/messageproperties/messages_en.properties

```diff
@@ -1302,7 +1302,7 @@ pihcore.uterineContractions=UC
 pihcore.weeks=weeks
 pihcore.new=New
 pihcore.followup=Follow-up
-
+pihcode.alert=You try to modify a patient who is not in your location
```

Comment:
Small typo, this should be "pihcore", not "pihcode"

Reply:
Also, defer to you all on the message content, but for the English translation, "You are trying to modify a patient who is not in your location" sounds better grammatically to me.